tests/test_overrides_parser.py

```diff
@@ -955,8 +961,9 @@ def test_get_key_element(override: str, expected: str) -> None:
         param("key={a:10,b:20}", "{a: 10, b: 20}", True, id="dict"),
         param("key={a:10,b:[1,2,3]}", "{a: 10, b: [1, 2, 3]}", True, id="dict"),
         param(
-            "key={/-\\+.$%*@: 1}",
-            "{/-\\\\+.$%*@: 1}",  # note that \ gets escaped
+            "key={%s: 1}" % UNQUOTED_SPECIAL,
+            # Note that \ gets escaped.
+            "{%s: 1}" % UNQUOTED_SPECIAL.replace("\\", "\\\\"),
```

Comment:
Can you explain what's going on here?

Reply:
> Can you explain what's going on here?

The test is still the same as before, I just wrote it based on`UNQUOTED_SPECIAL` (the motivation for introducing `UNQUOTED_SPECIAL` was to avoid having to modify multiple tests every time we'd add support for a new character in unquoted strings).

If you're asking why `\` gets escaped, this is because `_get_value_element_as_str()` calls `escape_special_characters()`.